File: hailiao_flutter/lib/models/content_audit_dto.dart
class ContentAuditDTO {
  final int? id;
  final int? contentType;
  final String? contentTypeLabel;
  final int? targetId;
  final String? content;
  final int? userId;
  final int? aiResult;
  final String? aiResultLabel;
  final int? aiScore;
  final int? manualResult;
  final String? manualResultLabel;
  final int? handlerId;
  final String? handleNote;
  final int? status;
  final String? statusLabel;
  final String? finalResultLabel;
  final String? createdAt;
  final String? handledAt;

  ContentAuditDTO({
    this.id,
    this.contentType,
    this.contentTypeLabel,
    this.targetId,
    this.content,
    this.userId,
    this.aiResult,
    this.aiResultLabel,
    this.aiScore,
    this.manualResult,
    this.manualResultLabel,
    this.handlerId,
    this.handleNote,
    this.status,
    this.statusLabel,
    this.finalResultLabel,
    this.createdAt,
    this.handledAt,
  });

  factory ContentAuditDTO.fromJson(Map<String, dynamic> json) {
    return ContentAuditDTO(
      id: (json['id'] as num?)?.toInt(),
      contentType: (json['contentType'] as num?)?.toInt(),
      contentTypeLabel: json['contentTypeLabel'] as String?,
      targetId: (json['targetId'] as num?)?.toInt(),
      content: json['content'] as String?,
      userId: (json['userId'] as num?)?.toInt(),
      aiResult: (json['aiResult'] as num?)?.toInt(),
      aiResultLabel: json['aiResultLabel'] as String?,
      aiScore: (json['aiScore'] as num?)?.toInt(),
      manualResult: (json['manualResult'] as num?)?.toInt(),
      manualResultLabel: json['manualResultLabel'] as String?,
      handlerId: (json['handlerId'] as num?)?.toInt(),
      handleNote: json['handleNote'] as String?,
      status: (json['status'] as num?)?.toInt(),
      statusLabel: json['statusLabel'] as String?,
      finalResultLabel: json['finalResultLabel'] as String?,
      createdAt: json['createdAt'] as String?,
      handledAt: json['handledAt'] as String?,
    );
  }
}

File: hailiao-api/src/main/java/com/hailiao/api/dto/ContentAuditDTO.java
package com.hailiao.api.dto;

import java.util.Date;

public class ContentAuditDTO {
    private Long id;
    private Integer contentType;
    private String contentTypeLabel;
    private Long targetId;
    private String content;
    private Long userId;
    private Integer aiResult;
    private String aiResultLabel;
    private Integer aiScore;
    private Integer manualResult;
    private String manualResultLabel;
    private Long handlerId;
    private String handleNote;
    private Integer status;
    private String statusLabel;
    private String finalResultLabel;
    private Date createdAt;
    private Date handledAt;

    public Long getId() { return id; }
    public void setId(Long id) { this.id = id; }
    public Integer getContentType() { return contentType; }
    public void setContentType(Integer contentType) { this.contentType = contentType; }
    public String getContentTypeLabel() { return contentTypeLabel; }
    public void setContentTypeLabel(String contentTypeLabel) { this.contentTypeLabel = contentTypeLabel; }
    public Long getTargetId() { return targetId; }
    public void setTargetId(Long targetId) { this.targetId = targetId; }
    public String getContent() { return content; }
    public void setContent(String content) { this.content = content; }
    public Long getUserId() { return userId; }
    public void setUserId(Long userId) { this.userId = userId; }
    public Integer getAiResult() { return aiResult; }
    public void setAiResult(Integer aiResult) { this.aiResult = aiResult; }
    public String getAiResultLabel() { return aiResultLabel; }
    public void setAiResultLabel(String aiResultLabel) { this.aiResultLabel = aiResultLabel; }
    public Integer getAiScore() { return aiScore; }
    public void setAiScore(Integer aiScore) { this.aiScore = aiScore; }
    public Integer getManualResult() { return manualResult; }
    public void setManualResult(Integer manualResult) { this.manualResult = manualResult; }
    public String getManualResultLabel() { return manualResultLabel; }
    public void setManualResultLabel(String manualResultLabel) { this.manualResultLabel = manualResultLabel; }
    public Long getHandlerId() { return handlerId; }
    public void setHandlerId(Long handlerId) { this.handlerId = handlerId; }
    public String getHandleNote() { return handleNote; }
    public void setHandleNote(String handleNote) { this.handleNote = handleNote; }
    public Integer getStatus() { return status; }
    public void setStatus(Integer status) { this.status = status; }
    public String getStatusLabel() { return statusLabel; }
    public void setStatusLabel(String statusLabel) { this.statusLabel = statusLabel; }
    public String getFinalResultLabel() { return finalResultLabel; }
    public void setFinalResultLabel(String finalResultLabel) { this.finalResultLabel = finalResultLabel; }
    public Date getCreatedAt() { return createdAt; }
    public void setCreatedAt(Date createdAt) { this.createdAt = createdAt; }
    public Date getHandledAt() { return handledAt; }
    public void setHandledAt(Date handledAt) { this.handledAt = handledAt; }
}
